modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v2_0/StrictWFS_2_0_Strategy.java

```diff
@@ -662,24 +665,48 @@ protected OperationType getOperationMetadata(final WFSOperationType operation) {
         return ftypeCrss;
     }
 
+    /**
+     * Returns parameters defined in OperationsMetadata
+     *
+     * @param parameterName
+     * @return
+     */
+    @SuppressWarnings("unchecked")
+    private Set<String> findParameters(final String parameterName) {
+        final OperationsMetadataType operationsMetadata = capabilities.getOperationsMetadata();
+
+        List<DomainType> parameters = operationsMetadata.getParameter();
+        for (DomainType parameter : parameters) {
+            if (parameterName.equals(parameter.getName())) {
+                Set<String> outputFormats = new HashSet<>();
```

Comment:
```suggestion
                Set<String> values = new HashSet<>();
```

The method is generic, we don't know they are output formats here.

Reply:
Oh, didn't notice there was a suggestion. I used foundValues.